server/http/helloapp/n_user_joined_channel.go

```diff
@@ -3,12 +3,11 @@ package helloapp
 import (
 	"net/http"
 
-	"github.com/mattermost/mattermost-plugin-apps/server/api"
 	"github.com/mattermost/mattermost-plugin-apps/server/apps"
 )
 
-func (h *helloapp) handleUserJoinedChannel(w http.ResponseWriter, req *http.Request,
-	claims *apps.JWTClaims, n *api.Notification) (int, error) {
+func (h *helloapp) nUserJoinedChannel(w http.ResponseWriter, req *http.Request,
```

Comment:
It's pretty intuitive using "f_xx", but I'm not sure "n_" is.  just a note and that we might consider.  Non-blocking

Reply:
these are "functions" and "notifications", probably worth expanding a bit 👍

Reply:
Added a comment